tests/unit/test_stages.py

```diff
@@ -1,28 +1,32 @@
+import itertools
 from unittest import TestCase
 
 import ddt
-import itertools
 import mock
 from xblock.field_data import DictFieldData
 from xblock.validation import ValidationMessage
 
 from group_project_v2.group_project import GroupActivityXBlock
-from group_project_v2.project_api import ProjectAPI
-from group_project_v2.stage import EvaluationDisplayStage, GradeDisplayStage, TeamEvaluationStage, PeerReviewStage
+from group_project_v2.project_api import TypedProjectAPI
+from group_project_v2.project_api.dtos import WorkgroupDetails, ReducedUserDetails, CompletionDetails
+from group_project_v2.stage import EvaluationDisplayStage, GradeDisplayStage, TeamEvaluationStage, PeerReviewStage, \
+    BaseGroupActivityStage, SubmissionStage
+from group_project_v2.stage.mixins import SimpleCompletionStageMixin
 from group_project_v2.stage.utils import ReviewState, StageState
 from group_project_v2.stage_components import PeerSelectorXBlock, GroupProjectReviewQuestionXBlock, GroupSelectorXBlock
-from tests.utils import TestWithPatchesMixin, make_review_item
+from tests.utils import TestWithPatchesMixin, make_review_item as mri, make_question, make_workgroup as mk_wg
```

Comment:
@bradenmacdonald only used in `ddt.data` test variants, and without these they become HUGE. They are already huge, but with full names it would be about twice as much. I believe this is a lesser evil :)